clang-tools-extra/clang-doc/HTMLGenerator.cpp

```diff
@@ -56,7 +56,7 @@ class HTMLTag {
   operator bool() = delete;
 
   bool IsSelfClosing() const;
-  llvm::SmallString<16> ToString() const;
+  const char* toString() const;
```

Comment:
Maybe this should be renamed to `c_str()` or `str()`?

Reply:
I think it be more appropriate to rename it c_str

Reply:
done